.travis.yml

```diff
@@ -14,6 +14,11 @@ install: travis_retry composer install --prefer-source
 
 script: composer ci
 
+after_success:
+  - vendor/bin/phpunit --coverage-clover coverage.clover
```

Comment:
after_success is executed for each build AFAIK. If not, I've been incorrectly understanding that for quite some time